pom.xml

```diff
@@ -1587,40 +1589,69 @@
       </build>
     </profile>
 
+    <!-- hudi-spark2-bundle is an alias for hudi-spark-bundle -->
```

Comment:
Should we rather affiliate the profiles, rather than artifacts here? It's not obvious and therefore confusing how spark profile trickle down into the artifacts

Reply:
Right.  I make it more readable now.